.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Drafter
+
+on:
+  workflow_dispatch:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - main
+  # pull_request event is required only for autolabeler
+  pull_request:
```

Comment:
This workflow seems to be only for release-drafter so we could remove all mention to autolabeler and only run this workflow on dispatch event or on main branch change

Reply:
I got confused by https://github.com/github/issue-labeler